src/command_modules/azure-cli-rdbms/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+0.2.5
```

Comment:
non-blocking question: is this feature still in the preview?

Reply:
No it is not , There is a plan to change the version soon , @QingqingYuan  would be the right person to answer that question.